packages/kit/src/views/Market/MarketDetailV2/components/SwapPanel/components/TokenInputSection/TokenList.tsx

```
@@ -100,14 +100,20 @@ export function TokenList({
-  });
+  const displayTokens = tokens
+    .map((token) => {
+      const tokenWithDetail = tokensWithDetails.result?.find(
+        (detailToken) =>
+          detailToken.networkId === token.networkId &&
+          detailToken.contractAddress === token.contractAddress,
+      );
+      return { ...token, ...tokenWithDetail };
+    })
+    .sort((a, b) => {
+      const valueA = parseFloat(a.valueProps?.value || '0');
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)